### PR TITLE
Portfolio: validate contact form PR

### PR DESCRIPTION
**Pull request by normainobary2021**
In this PR, I added the following changes:
- Included javascript code for email validation in main.js
- Included styling for span element font color in style.css
- Added span tag to index.html to show error message upon validation check.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @normainobary2021, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that should be created using peer-programming , all students in peer-programing group should consider contributing to this PR 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by normainobary2021** [comment]
Thank you for the reminder @Ismailco. I have obliged and resubmitted for review. 👍

File: index.html [hunk 1]
```diff
@@ -166,17 +166,18 @@ <h1 class="primary-text-change">About<br>Myself</h1>
             </ul>
         </div>
     </section>
-    <form id="contact-form" class="contact-form" action="https://formspree.io/f/xoqrojoe" method="post">
+    <form novalidate id="contact-form" class="contact-form" action="https://formspree.io/f/xoqrojoe" method="post">
```
Comment (wuletawwonte) [inline index.html:169]: - [ ] [OPTIONAL] Please remove `novalidate` property from the form. I think it should validate for empty fields and some basic validations. You are requested to add extra custom validation for the email field.

**Review by wuletawwonte — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @normainobary2021 & @haftamudesta  👋🏿👋🏿👋🏿, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

approve

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (normainobary2021) [reply index.html:169]: Thank you very much for the kind review. I will definitely remove the novalidate, I was having a challenge with the javascript validation getting overshadowed by the html5 one though, it seems to take precedence so I couldn't see my javascript error message. What can I do for both to show up together?

**Pull request merged**